### PR TITLE
Solve 06 tasks and fix PR comments

### PR DESCRIPTION
**Pull request by dariadm**
Solve 06_01 and 06_02 tasks
Fix 06_02 task test file
Fix previous PR comments

### CODE REVIEW

**Comment by jsru** [comment]
Добавляю преподавателя (@dosandk) для код-ревью.

File: 05-dom-document-loading/1-notification/index.js [hunk 1]
```diff
@@ -40,17 +40,12 @@ export default class NotificationMessage {
     }
   }
 
-  show (div) {
+  show (div = document.body) {
```
Comment (dosandk) [inline 05-dom-document-loading/1-notification/index.js:43]: 👍 good

File: 04-oop-basic-intro-to-dom/1-column-chart/index.js [hunk 2]
```diff
@@ -41,15 +41,15 @@ export default class ColumnChart {
 
     this.element = element.firstElementChild;
 
-    if (this.data.length == 0) {
+    if (!this.data.length) {
```
Comment (dosandk) [inline 04-oop-basic-intro-to-dom/1-column-chart/index.js:44]: 👍 good

File: 06-events-practice/2-tooltip/index.js [hunk 3]
```diff
@@ -1,5 +1,72 @@
 class Tooltip {
+  static instance = null;
 
+  onPointerover = (event) => {
+    if (event.target.dataset.tooltip) {
+      this.render(event.target.dataset.tooltip);
+
+      document.addEventListener('pointermove', this.onPointermove);
```
Comment (dosandk) [inline 06-events-practice/2-tooltip/index.js:8]: 👍 good

**Review by dosandk — APPROVED** [review]
good 👍

**Pull request merged**